units/crons.go

```diff
@@ -484,6 +484,32 @@ func PopulateSchedulerJobs(env evergreen.Environment) amboy.QueueOperation {
 		lastRuntime, err := model.FindTaskQueueGenerationRuntime()
 		catcher.Add(err)
 
+		// clear queues for purposefully disabled distros
+		disabledDistros, err := distro.Find(distro.ByIsDisabled(env.Settings().ContainerPools.Pools))
+		catcher.Add(err)
```

Comment:
I think you probably want to bail immediately if there is an error instead of adding it to a catcher.

Reply:
I would prefer we continue on error like `ByNeedsPlannin`. If there's a problem with the task-clearing logic and we return on error, it'll prevent scheduler jobs from being queued (below this).

Reply:
I'm modeling off of `ByNeedsPlanning` below, which also adds to a catcher